conda_pypi/cli/pypi.py

```diff
@@ -0,0 +1,196 @@
+"""
+conda pypi subcommand for CLI
+"""
+
+from __future__ import annotations
+
+import argparse
+from pathlib import Path
+from logging import getLogger
+
+from conda.auxlib.ish import dals
+from conda.cli.conda_argparse import (
+    add_output_and_prompt_options,
+    add_parser_help,
+    add_parser_prefix,
+    _GreedySubParsersAction,
+)
+from conda.exceptions import ArgumentError
+from conda.base.context import context
+from conda.models.match_spec import MatchSpec
+
+from conda_pypi import convert_tree
+
+
+logger = getLogger(__name__)
+
+
+def configure_parser(parser: argparse.ArgumentParser):
+    add_parser_help(parser)
+    # This adds --prefix/--name mutually exclusive options
+    add_parser_prefix(parser)
+    add_output_and_prompt_options(parser)
+
+    sub_parsers = parser.add_subparsers(
+        metavar="COMMAND",
+        title="commands",
+        description="The following subcommands are available.",
+        dest="cmd",
+        action=_GreedySubParsersAction,
+        required=True,
+    )
+
+    # install subcommand
+    summary = "Install PyPI packages as conda packages"
+    description = summary
+    epilog = dals(
+        """
+
+        Install PyPI packages as conda packages.  Any dependencies that are
+        available on the configured conda channels will be installed with `conda`,
+        while the rest will be converted to conda packages from PyPI.
+
+        Examples:
+
+        Install a single PyPI package into the current conda environment::
+
+            conda pypi install requests
+
+        Install multiple PyPI packages with specific versions::
+
+            conda pypi install "numpy>=1.20" "pandas==1.5.0"
+
+        Install packages into a specific conda environment::
+
+            conda pypi install -n myenv flask django
+
+        Install packages using only PyPI (skip configured conda channels)::
+
+            conda pypi install --override-channels fastapi
+
+        """
+    )
+    install = sub_parsers.add_parser(
+        "install",
+        help=summary,
+        description=description,
+        epilog=epilog,
+    )
+    install.add_argument(
+        "--override-channels",
+        action="store_true",
+        help="Do not search default or .condarc channels. Will search pypi.",
+    )
+    install.add_argument(
+        "packages",
+        metavar="PACKAGE",
+        nargs="*",
+        help="PyPI packages to install",
+    )
+
```

Comment:
Out of curiosity, if we follow the pattern in conda to define the `func` per subparser would that skip the `execute` function below?

```suggestion
install.set_defaults(func="conda_pypi.cli.pypi.execute_install")
```